.travis.yml

```diff
@@ -1,6 +1,9 @@
 sudo: required
 dist: trusty
 
+notifications:
+  email: false
```

Comment:
A ton, basically every single time a new branch is built, whenever the build breaks, and whenever the build gets fixed.